pkgs/development/python-modules/camset/default.nix

```
@@ -0,0 +1,41 @@
+{
+  python312Packages,
+  fetchFromGitHub,
+  gobject-introspection,
+  v4l-utils,
+  wrapGAppsHook3,
+  lib
+}:
+
+python312Packages.buildPythonPackage rec {
+  name = "camset";
+  version = "0.0.1";
+  src = fetchFromGitHub {
```

```suggestion
  pname = "camset";
  version = "0.0.1";
  pyproject = true;

  src = fetchFromGitHub {
```
Where do you take the version from? Please read https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning.